### PR TITLE
29308 css length units in svg

### PR DESCRIPTION
**Pull request by dletorey**
### Description

- Added an example of of using level 3 CSS Length units to `svg` element in the `height` & `width` attributes.
- Added related relase not to 119

### Motivation

Working on [issue 29308](https://github.com/mdn/content/issues/29308)

### Related issues and pull requests

- [Bugzilla 1287054](https://bugzilla.mozilla.org/show_bug.cgi?id=1287054)

### CODE REVIEW

File: files/en-us/web/svg/element/svg/index.md [hunk 1]
```diff
@@ -36,7 +40,54 @@ svg {
 </svg>
 ```
 
-{{EmbedLiveSample('Example', 300, 100)}}
+{{EmbedLiveSample('nested_svg-elements', 300, 100)}}
+
+### Using dynamic viewport lengths
+
+This example the `height` and `width` attributes, on the `svg` element, are set using the dynamic viewport values `vmin`
```
Comment (zfox23) [inline files/en-us/web/svg/element/svg/index.md:47]: One suggestion in this text:
```suggestion
In this example, the `height` and `width` attributes on the `svg` element are set using the dynamic viewport value `60vmin`, equivalent to 60% of the viewport's width or height, whichever is smaller. 
```

Secondly, I like your comment about changing the iframe's dimensions to support the example. I think more people would read that text if it were up here, on a new line underneath _this_ text, such as:

>To change the iframe viewport's dimensions - thus impacting the dimensions of the `svg` - click and drag the iframe's dotted red border from its bottom right corner.

File: files/en-us/web/svg/element/svg/index.md [hunk 2]
```diff
@@ -36,7 +40,54 @@ svg {
 </svg>
 ```
 
-{{EmbedLiveSample('Example', 300, 100)}}
+{{EmbedLiveSample('nested_svg-elements', 300, 100)}}
+
+### Using dynamic viewport lengths
```
Comment (zfox23) [inline files/en-us/web/svg/element/svg/index.md:45]:
```suggestion
### Using dynamic viewport units
```

**Review by zfox23 — CHANGES_REQUESTED** [review]
thank you :)

**Comment by longsonr** [comment]
I'm afraid this isn't really right. Firefox has supported CSS level 3 units on `<svg>` and some other elements such as `<rect>`, `<circle>`, `<ellipse>`, `<image>`, `<foreignObject>` and `<use>` for some time. What the bug does is support those units on all elements that were not previously supported for example `<line>`, `<filter>`, `<clipPath>` (and others). And it's not limited to width/height either, you can write `<line x1="10ch" y1="10vw" x2="20rem" y2="15vmin"/>` for example.

**Comment by longsonr** [comment]
Oh and a level 3 length is anything in https://drafts.csswg.org/css-values-3/#viewport-relative-lengths, as opposed to a level 4 length which would be from the css-values-4 spec instead. level 4 is a superset of level 3.

**Comment by dletorey** [comment]
> I'm afraid this isn't really right. Firefox has supported CSS level 3 units on `<svg>` and some other elements such as `<rect>`, `<circle>`, `<ellipse>`, `<image>`, `<foreignObject>` and `<use>` for some time. What the bug does is support those units on all elements that were not previously supported for example `<line>`, `<filter>`, `<clipPath>` (and others). And it's not limited to width/height either, you can write `<line x1="10ch" y1="10vw" x2="20rem" y2="15vmin"/>` for example.

Understood, working on it

**Review by zfox23 — CHANGES_REQUESTED** [review]
one grammar nit, and one suggestion for brevity

**Review by zfox23 — APPROVED** [review]
thank you!

**Comment by dletorey** [comment]
> thank you!

That was super quick

**Comment by zfox23** [comment]
@longsonr i want to make sure that your comments are addressed. if you see this and are comfortable with the changes, please merge. otherwise, i'll merge tomorrow evening ☺️

**Comment by longsonr** [comment]
This looks great. I'm happy with all changes.

**Pull request merged**